src/src/Entity/FormEntity.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FormEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: FormEntityRepository::class)]
+class FormEntity
```

Comment:
I would prefer to name this to convey the meaning of the entity (so, e.g., `Rating` or `Feedback`), not the "form" it takes (the pun is absolutely intended :)).